File: etl/models/partido.py
from sqlalchemy import Column, Integer, String 
from .base import Base

class Partido(Base):
    __tablename__ = "partido"

    nr_partido = Column(Integer, primary_key=True)
    sg_partido = Column(String(20), nullable=False)
    nm_partido = Column(String, nullable=False)

File: src/backend/VoteBem/Entities/Partido.cs
namespace VoteBem.Entities
{
    public class Partido
    {
        public int NrPartido { get; private set; }
        public string SgPartido { get; private set; } = null!;
        public string NmPartido { get; private set; } = null!;

        public ICollection<Candidatura> Candidaturas { get; private set; } = [];

        protected Partido() { }
    }
}
